Sources/LibAuk/Storage/SecureStorage.swift

```diff
@@ -49,7 +50,7 @@ class SecureStorage: SecureStorageProtocol {
         self.keychain = keychain
     }
     
-    func createKey(name: String) -> AnyPublisher<Void, Error> {
+    func createKey(password: String?, name: String) -> AnyPublisher<Void, Error> {
```

Comment:
Let's make this function argument overloading by having the password default value to be empty. Also, let's call it `passphrase` instead of `password`.